spec_5/index.js

```diff
@@ -0,0 +1,8 @@
+
+const disemvowel = phrase => {
+  return phrase.replace( /[aeiou\s]/g, "" )
+}
+
+console.log(
```

Comment:
I would omit logs from PRs. Did you write tests for this? Tests would make the test-case log unnecessary. Also, what about phrases with apostrophe's (AKA: sky-commas)? What about if someone capitalizes letters? When I tried calling disemvowel('I thInk thAt ThIS WOn't work) it broke.